spec/features/user_visits_homepage_spec.rb

```diff
@@ -0,0 +1,9 @@
+require "rails_helper"
+
+feature 'User visits homepage' do
+  scenario 'successfully' do
+    visit root_path
+
+    expect(page).to have_css('h1', text: 'Library Catalog')
```

Comment:
You may want to add a class instead of testing against a tag. If you test against a tag and there is another `h1` you could have ambiguous results. Maybe something like this `have_css('.page-title', text: 'Library Catalog')`